torch/onnx/utils.py

```diff
@@ -750,6 +745,12 @@ def _export(model, args, f, export_params=True, verbose=False, training=None,
                         opened_file.write(v)
             else:
                 raise RuntimeError("Unknown export type")
+
+            if operator_export_type is OperatorExportTypes.ONNX and \
+               not val_use_external_data_format:
+                # Only run checker if we are using ONNX export type and
```

Comment:
I know you're just moving this, but let's improve the code while we're at it:
* use parentheses instead of backslash
* move the comment above the if
* change the comment to explain why we don't run the check with `val_use_external_data_format`